cypress/integration/smoke/tx_history.spec.js

```diff
@@ -0,0 +1,234 @@
+const SAFE = 'rin:0x87a57cBf742CC1Fc702D0E9BF595b1E056693e2f'
+
+// Logo pathnames
+const CONTRACT_INTERACTION = '/app/static/media/custom.6f7b401c.svg'
+const WRAPPED_ETH = '/tokens/logos/0xc778417E063141139Fce010982780140Aa0cD5Ab.png'
+const MULTI_SEND_CONTRACT = '/contracts/logos/0x40A2aCCbd92BCA938b02010E17A5b8929b49130D.png'
+const OUTGOING = '/app/static/media/outgoing.d744cc02.svg'
+const ETH = '/chains/4/currency_logo.png'
+const INCOMING = '/app/static/media/incoming.1bf5be26.svg'
+const NFT = '/tokens/logos/0x4DBCdF9B62e891a7cec5A2568C3F4FAF9E8Abe2b.png'
+const USDC = '/app/static/media/incoming.1bf5be26.svg'
+
+describe('Dashboard', () => {
+  before(() => {
+    // Go to the test Safe transaction history
+    cy.visit(`/${SAFE}/transactions/history`)
+    cy.contains('a', 'Accept selection').click()
+  })
+
+  it('should display June 9th transactions', () => {
+    const DATE = 'Jun 9, 2022'
+
+    // Date label
+    cy.contains('p', DATE).should('exist')
+
+    // Transaction summaries
+    const accordions = cy.contains('p', DATE).next().children()
```

Comment:
You mean rows?

Reply:
They are MUI Accordions. I'll change it to be more generic though.